user-manager-service/src/main/resources/db/changelog/2023.11_initial-users-table.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                     http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="create_users_table" author="Sasha_M">
+        <createTable  tableName="users">
+            <column name="id" type="bigint">
+                <constraints primaryKey="true" nullable="false"/>
```

Comment:
maybe nullable redundant, because primary key consists nullable(false) and unique(true).

Reply:
I've tested it, and yes it is redundant. Interesting cause in most tutorials/guides nullable="false" specified)